test/module/irohad/torii/torii_queries_test.cpp

```diff
@@ -431,8 +454,9 @@ TEST_F(ToriiQueriesTest, FindSignatoriesHasRolePermissions) {
   // TODO: refactor this to use stateful validation mocks
   iroha::pubkey_t pubkey;
   std::fill(pubkey.begin(), pubkey.end(), 0x1);
-  std::vector<iroha::pubkey_t> keys;
-  keys.push_back(pubkey);
+  std::vector<shared_model::interface::types::PubkeyType> keys;
```

Comment:
You can use initialiser list

Reply:
I need to initialize pubkey with std::fill before placing it inside a vector, so initializer list is not an option